### PR TITLE
The header is narrower now

### PR DESCRIPTION
**Pull request by leilasadr**

### CODE REVIEW

File: src/components/Header/Header.css
Comment (min-dles) [inline src/components/Header/Header.css:21]: should be able to merge this to main if you can add the z-index back! without it, i think the buttons float over the header when users scroll

**Review by min-dles — CHANGES_REQUESTED** [review]
z-index back in Header.css, or another fix that would prevent buttons from floating over the header nav when user scrolls

**Comment by Keegan-Odell** [comment]
I dealt with the merge conflict in overview...so we are good on that front...once the css is ready to go we can merge

**Pull request merged**
